ruleset/rules/0265-php_rules.xml

```diff
@@ -57,12 +51,12 @@
   <rule id="31411" level="6">
     <if_sid>31410</if_sid>
     <match> expects parameter 1 to be string, array given in</match>
-    <group>attack,pci_dss_6.5,pci_dss_11.4,gdpr_IV_35.7.d,nist_800_53_SA.11,nist_800_53_SI.4,tsc_CC6.6,tsc_CC7.1,tsc_CC8.1,tsc_CC6.1,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,</group>
     <description>PHP web attack.</description>
     <mitre>
       <id>T1190</id>
-      <id>T107.001</id>
+      <id>T1071.001</id>
     </mitre>
+    <group>attack,pci_dss_6.5,pci_dss_11.4,gdpr_IV_35.7.d,nist_800_53_SA.11,nist_800_53_SI.4,tsc_CC6.6,tsc_CC7.1,tsc_CC8.1,tsc_CC6.1,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,</group>
```

Comment:
Groups are not correctly ordered

Reply:
I think there is a reason for this as PCI is considered more important than GDPR and GDPR is mroe important than nist and so on. All the rules in ruleset more or less respect ths order, I doubt it's random. Perhaps we should remove this rule from the criteria

Reply:
begrudgingly done